ocs_ci/helpers/helpers.py

```diff
@@ -1546,6 +1546,12 @@ def measure_pvc_creation_time_bulk(interface, pvc_name_list, wait_time=60):
             start = [i for i in logs if re.search(f"provision.*{name}.*started", i)]
             end = [i for i in logs if re.search(f"provision.*{name}.*succeeded", i)]
             if not start or not end:
+                logging.info(f"PVC {name} missing start or end time")
+                try:
+                    logging.info(f"PVC {name} creation start {start}")
+                    logging.info(f"PVC {name} creation complete {end}")
+                except UnexpectedBehaviour:
+                    logging.warning(f"PVC {name} creation data missing")
```

Comment:
What would raise `UnexpectedBehaviour` exception in the try block? At first sight, I don't see anything which would raise it. Maybe we just want to log this info without having the try except block in the first place.

Reply:
@mbukatov we are observing logs missing from CSI when we do bulk of PVC creation and deletion. adding this log will help in to understand if any of the value/log is not present in the CSI logs.
So expectation behind the exception is either `start` or `end` will get null value and want to display this warning before failing the test.

Reply:
We are suspecting CSI pod logs are getting cleaned up after a certain point, i.e. if we do create 1500 PVC creation and deletion 4 times CSI logs will be loaded, it can hold only particular size and further it's getting flushed (i.e. it's kind of all logs lost/missing), planning to use logging to resolve this problem but still in exploratory stage.